app/src/main/res/layout/member_fragment.xml

```diff
@@ -42,7 +42,7 @@
 
         <TextView
             android:id="@+id/member_name"
-            android:layout_width="wrap_content"
+            android:layout_width="match_parent"
```

Comment:
why?

Reply:
when i use wrap content it crosses the boarders of the screen. and to make it as parent wide solved it. i change it now to 0dp, and added a constraint to the end, it is in the next commit